pkg/modulereader/resreader.go

```diff
@@ -39,16 +41,41 @@ type VarInfo struct {
 	Required    bool
 }
 
+// OutputInfo stores information about module output values
+type OutputInfo struct {
+	Name        string
+	Description string   `yaml:",omitempty"`
+	Sensitive   bool     `yaml:",omitempty"`
+	DependsOn   []string `yaml:"depends_on,omitempty"`
```

Comment:
IIUC `DependsOn` isn't functionally supported yet.  Shall we omit it from yaml?

Reply:
+1